### PR TITLE
NOISSUE - Add Groups CLI Documentation

### PR DESCRIPTION
**Pull request by rodneyosodo**
### What does this do?
Add CLI documentation for groups

### Which issue(s) does this PR fix/relate to?
N/A

### List any changes that modify/break current functionality
N/A

### Have you included tests for your changes?
No

### Did you document any new/modified functionality?
Yes

### Notes
N/A

### CODE REVIEW

**Review by drasko — DISMISSED** [review]
LGTM

File: docs/cli.md [hunk 1]
```diff
@@ -365,3 +365,49 @@ mainflux-cli keys retrieve <key_id> <user_token>
 ```bash
 mainflux-cli keys revoke <key_id> <user_token>
 ```
+
+### Groups
+
+#### Create Group
+```bash
+mainflux-cli groups create '{"Name":<group_name>,"Description":<description>,"ParentID":<parent_id>,"Metadata":<metadata>}' <user_token>
```
Comment (dborovcanin) [inline docs/cli.md]: Put placeholders under quotation marks like `"<placeholder>"` to make it valid JSON.

**Review by dborovcanin — CHANGES_REQUESTED** [review]

File: docs/cli.md (hunk 1 above)
Comment (rodneyosodo) [reply docs/cli.md]: Done

File: docs/cli.md [hunk 2]
```diff
@@ -411,3 +411,49 @@ mainflux-cli groups delete <group_id> <user_token>
...
+mainflux-cli groups unassign <member_ids> <group_id>  <user_token>
+```
+#### Delete Group
+```bash
+mainflux-cli groups delete <group_id> <user_token>
+```
+
+#### Bootstrap configuration
+```bash
+mainflux-cli bootstrap bootstrap <external_id> <external_key>
```
Comment (manuio) [inline docs/cli.md]: Duplication from line 412

**Review by manuio — CHANGES_REQUESTED** [review]

File: docs/cli.md (hunk 2 above)
Comment (rodneyosodo) [reply docs/cli.md]: Removed

**Review by manuio — APPROVED** [review]
LGTM

**Review by drasko — APPROVED** [review]
LGTM

**Review by dborovcanin — APPROVED** [review]

**Comment by fbugarski** [comment]
LGTM

**Pull request merged**